e2e/consecutive-changes/rector.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\CodingStyle\Rector\Stmt\NewlineAfterStatementRector;
+use Rector\Config\RectorConfig;
+use Rector\EarlyReturn\Rector\If_\ChangeAndIfToEarlyReturnRector;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->paths([
+        __DIR__ . '/src',
+    ]);
+
```

Comment:
FileCacheStorage need to be used like in consecutive dry run before 

https://github.com/rectorphp/rector-src/blob/dc57cffad71a875b8ccd11be265260754c47d32e/e2e/applied-rule-removed-node-with-cache/rector.php#L11

Reply:
good catch!

Reply:
I suffixed the folder with `with-cache` to be expliit